src/qtism/data/storage/xml/marshalling/MatchInteractionMarshaller.php

```diff
@@ -70,11 +78,20 @@ protected function unmarshallChildrenKnown(DOMElement $element, QtiComponentColl
                 $component->setMaxAssociations($maxAssociations);
             } elseif (Version::compare($version, '2.1.0', '<') === true) {
                 $msg = "The mandatory attribute 'maxAssociations' is missing from the 'matchInteraction' element.";
+
                 throw new UnmarshallingException($msg, $element);
             }
 
             // minAssociations.
-            if (Version::compare($version, '2.1.0', '>=') === true && ($minAssociations = $this->getDOMElementAttributeAs($element, 'minAssociations', 'integer')) !== null) {
+            if (Version::compare(
+                    $version,
+                    '2.1.0',
+                    '>='
+                ) === true && ($minAssociations = $this->getDOMElementAttributeAs(
+                    $element,
+                    'minAssociations',
+                    'integer'
+                )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place